deltachat-jsonrpc/.gitignore

```diff
@@ -0,0 +1,6 @@
+/target
+/accounts
+
+types.ts
+
+.cargo
```

Comment:
Is it possible putting a .gitignore file into a subfolder?

Reply:
yes absolutely, tried it many times before

Reply:
Nice, didn't know